docs/local_install_skaffold.md

```diff
@@ -1,51 +1,280 @@
-# Local install of Substra using kubernetes and skaffold
+# Local install of Substra using Kubernetes and Skaffold
+
+> This is an ongoing document, please feel free to reach us or to raise any issue.
+
+This guide will help you to run the Substra platform on your machine in development mode, with a two organisations setup.
```

Comment:
typo: organizations

Also, we don't use the word organization anywhere else, we tend to use "node".

Reply:
Noted!